packages/wrangler/src/index.ts

```diff
@@ -765,13 +765,17 @@ export async function main(argv: string[]): Promise<void> {
 		}
 		throw e;
 	} finally {
-		await closeSentry();
 		// In the bootstrapper script `bin/wrangler.js`, we open an IPC channel, so
 		// IPC messages from this process are propagated through the bootstrapper.
 		// Make sure this channel is closed once it's no longer needed, so we can
 		// cleanly exit. Note, we don't want to disconnect if this file was imported
 		// in Jest, as that would stop communication with the test runner.
 		if (typeof jest === "undefined") process.disconnect?.();
+
+		// Close Sentry after disconnecting the IPC channel. Doing this before leads
+		// to hanging `workerd` processes.
+		// TODO(soon): work out why
```

Comment:
I don't really rate this sort of TODO comment. They usually end up being lost in the future. How about we just create a ticket to track the work that needs to be done?